ui/js/controllers/main.js

```diff
@@ -546,6 +554,12 @@ treeherderApp.controller('MainCtrl', [
             return url;
         };
 
+        $scope.dropLocationSearchParam = function(param) {
```

Comment:
Note to self: If we ever wanted to use this for a url parameter that doesn't have a value assigned to it, this probably wouldn't work.